types/filestack-react/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for filestack-react 4.0
+// Project: https://github.com/filestack/filestack-react
+// Definitions by: Phil Ting <https://github.com/philting>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import * as client from 'filestack-js';
+// These two lines are not mergeable, and this error is considered a false positive
+//   in eslint: https://eslint.org/docs/latest/rules/no-duplicate-imports
+// tslint:disable-next-line:no-duplicate-imports
```

Comment:
```suggestion
export * as client from 'filestack-js';
```
(and you can remove last export)

Reply:
I've made the change. However, I'd like to point out that there is a [known issue in TSLint with the `whitespace` rule](https://github.com/microsoft/dtslint/issues/322), and this was the recommended workaround. Trying to disable the `whitespace` rule doesn't work either since it's just a parsing error.

You can see [in this build](https://github.com/DefinitelyTyped/DefinitelyTyped/runs/8280352959?check_suite_focus=true) that the CI passed even though there was a runtime error. I'm assuming that it's not that big of an issue since Prettier and `typedef-whitespace` should have enough coverage, and also that we're migrating to ESLint anyway. Just thought it's worth mentioning since I originally thought the runtime error would break the build.

@sheetalkamat FYI